pkg/agent/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -160,8 +160,10 @@ func NewNetworkPolicyController(antreaClientGetter agent.AntreaClientProvider,
 	}
 
 	// Use nodeName to filter resources when watching resources.
+	timeoutSeconds := int64(60)
 	options := metav1.ListOptions{
-		FieldSelector: fields.OneTermEqualSelector("nodeName", nodeName).String(),
+		FieldSelector:  fields.OneTermEqualSelector("nodeName", nodeName).String(),
+		TimeoutSeconds: &timeoutSeconds,
```

Comment:
This will change the watch request's behavior: when timeout is not set, server will generate a random timeout greater than 2 hour. After this change, it forces the watch request will restart every minute, which will cause great pressure to antrea-controller in scale cluster.

If it's just to reduce some connection error when AntreaProxy hasn't installed flows, I think it's acceptable. And ideally we could have some mechanism to synchronize AntreaProxy and other components' startup later.

Reply:
Ok. Reset. There'll be error logs like:
```
W0928 07:44:57.347928    3392 reflector.go:436] pkg/mod/k8s.io/client-go@v0.21.0/tools/cache/reflector.go:167: watch of *v1.ConfigMap ended with: an error on the server ("unable to decode an event from the watch stream: read tcp 10.96.0.1:61819->10.96.0.1:443: wsarecv: An established connection was aborted by the software in your host machine.") has prevented the request from succeeding
W0928 07:44:57.347928    3392 reflector.go:436] pkg/mod/k8s.io/client-go@v0.21.0/tools/cache/reflector.go:167: watch of *v1.ConfigMap ended with: an error on the server ("unable to decode an event from the watch stream: read tcp 10.96.0.1:61819->10.96.0.1:443: wsarecv: An established connection was aborted by the software in your host machine.") has prevented the request from succeeding
W0928 07:44:57.347928    3392 reflector.go:436] pkg/mod/k8s.io/client-go@v0.21.0/tools/cache/reflector.go:167: watch of *v1.ConfigMap ended with: an error on the server ("unable to decode an event from the watch stream: read tcp 10.96.0.1:61819->10.96.0.1:443: wsarecv: An established connection was aborted by the software in your host machine.") has prevented the request from succeedinga
```